services/utils.service.js

```diff
@@ -26,20 +26,28 @@ function loadSettings() {
     }
 }
 
-function getJiraBoardIdForTeam(client, teamName) {
-    return loadSettings().then(settings3 => {
-        let teams = settings3.clients[client].teams;
-        let team = teams.find(t => t.teamName == teamName)
-        return team != null ? team.jiraBoardId : null
-    });
+async function getJiraBoardIdForTeam(clientId, teamId) {
+    let client = getClient(clientId)
+    const client_1 = await Promise.resolve(client);
+    let team = client_1.teams.find(t => t.teamId == teamId);
+    return team != null ? team.jiraBoardId : null;
```

Comment:
you can use if(team) instead of if(team != null)

```suggestion
    return team ? team.jiraBoardId : null;
```